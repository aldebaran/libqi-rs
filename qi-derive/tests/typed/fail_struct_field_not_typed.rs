struct F;

#[derive(qi_derive::Typed)]
struct S {
    pub f: F,
}

fn main() {}
