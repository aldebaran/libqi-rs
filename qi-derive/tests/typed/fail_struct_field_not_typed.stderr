error[E0277]: the trait bound `F: Typed` is not satisfied
 --> tests/typed/fail_struct_field_not_typed.rs:5:12
  |
5 |     pub f: F,
  |            ^ the trait `Typed` is not implemented for `F`
  |
  = help: the following other types implement trait `Typed`:
            &str
            ()
            BTreeMap<K, V>
            HashMap<K, V>
            PhantomData<T>
            S
            Signature
            Vec<T>
          and $N others
